docker-test-framework/settings.xml

```diff
@@ -12,10 +12,6 @@ as the operative profile to be used in building and testing jargon against a cho
     <profile>
 			<id>flyway-docker</id>
 			<properties>
```

Comment:
Perhaps `<properties>` can be removed too.

Reply:
done